radix-engine/src/blueprints/epoch_manager/events/validator.rs

```diff
@@ -26,3 +26,27 @@ pub struct ClaimXrdEvent {
 pub struct UpdateAcceptingStakeDelegationStateEvent {
     pub accepts_delegation: bool,
 }
+
+#[derive(ScryptoSbor, ScryptoEvent, PartialEq, Eq, Debug)]
+pub struct RewardAppliedEvent {
+    /// An amount of XRD added to the validator's stake as a reward.
+    /// Note: this number represent a final gain of value of all stake units of the validator, which
+    /// means that any applicable reliability penalty and validator fee were already subtracted.
+    pub stake_added_xrd: Decimal,
+    /// A total supply of stake units of the validator at the moment of applying this reward.
+    /// Note: calculating `stake_added_xrd / liquidity_token_supply` gives a convenient "XRD reward
+    /// per stake unit" factor, which may be used to easily calculate individual staker's gains.
```

Comment:
_(note from author)_

@dhedey you asked for "XRD reward per stake unit" (at https://rdxworks.slack.com/archives/C01HK4QFXNY/p1683742513587619?thread_ts=1683570653.786939&cid=C01HK4QFXNY) and I decided to provide this information as 2 numbers, for extra details.